lib/storage/s3.js

```diff
@@ -59,7 +98,22 @@ module.exports = function() {
       }
       https = options.https;
       cachingTime = options.cachingTime;
-      return callback(null);
+      // CORS
+      client.getBucketCors(getCorsConfig, function (err, data) {
+        if (err) {// an error occurred
+          console.log(err, err.stack);
+          return callback(null);
+        } else { // successful response
+          if (data.CORSRules !== putCorsConfig.CORSConfiguration.CORSRules) {
```

Comment:
These are arrays, yes? The comparison will always fail because of the way JavaScript compares objects. The `isEqual` method of lodash might do what you want here.